assignment-section-05/classification_model/config.yml

```diff
@@ -0,0 +1,51 @@
+# Package Overview
+package_name: regression_model
+
+# Data Files
+raw_data_file: raw.csv
```

Comment:
where does this file come from? 

should we not include in the config the url to fetch the data instead ?